### PR TITLE
Add works section and about me

### PR DESCRIPTION
**Pull request by Matthieu96Code**
last part of the portfolio project_mobile first

- stick to the design as much as possible
- Seven pages in works sections
- About me section (use of ul and li)
- form
- and contact (use of ul, li, and link tab )

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,5 +61,313 @@ <h2 class="supporting-text">
                 </li>
             </ul>
         </section>
+        <section class="projects">
+            <h2>
+                My recent Works
+            </h2>
```
Comment (joskalenda) [inline index.html]: - [ ] Well done on creating the work section, but please update this section by making a few touches to match the original design font family, remember this comment does not only apply to this title, but to the whole app, you are using a different font family compared to the one in the original template.

File: index.html [hunk 2]
```diff
@@ -61,5 +61,313 @@ <h2 class="supporting-text">
...
+                    <li>
+                        <a href="#" >bootstrap</a>
+                    </li>
+                    <li>
+                        <a href="#" >Ruby</a>
+                    </li>
+                </ul>
+                <button>
+                    See Project
+                </button>
```
Comment (joskalenda) [inline index.html]: - [ ] Your button does not have any type, I would suggest you add one of type button for all of your `see project`'s buttons

File: index.html [hunk 3]
```diff
@@ -61,5 +61,313 @@ <h2 class="supporting-text">
...
+            </section>
+        </section>
+        <footer>
+            <div class="about-me">
+                <h1 class="about-me-title">
+                    About me
+                </h1>
+                <p>
```
Comment (joskalenda) [inline index.html]: - [ ] Consider changing this h1 into h2 - h6 instead, following the HTML best practice, a page should at least have one h1 tag.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Matthieu96Code , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Green linters
- [ ]... ✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Matthieu96Code, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**